src/opensles/AudioStreamOpenSLES.cpp

```diff
@@ -49,6 +49,29 @@ AudioStreamOpenSLES::~AudioStreamOpenSLES() {
     delete[] mCallbackBuffer;
 }
 
+#define AUDIO_CHANNEL_COUNT_MAX          30u
```

Comment:
Convention is to use `constexpr` and `kAudioChannelCountMax`

Reply:
The compiler rejected that. I'll try again.

Reply:
Done